src/Dev/Tasks/TagsToShortcodeHelper.php

```diff
@@ -210,7 +210,8 @@ private function getTagsInContent($content)
     {
         $resultTags = [];
 
-        preg_match_all('/<('.$this->validTagsPattern.').*?('.$this->validAttributesPattern.')\s*=.*?>/i', $content, $matches, PREG_SET_ORDER);
+        $regex = '/<('.$this->validTagsPattern.')\s[^>]*?('.$this->validAttributesPattern.')\s*=.*?>/i';
```

Comment:
This is where the magic happens. Replacing `.*?` for `\s[^>]*?` means that we will stop looking for an `href|src` attribute if our tag doesn't contain a space or if we hit a `>` character.

That way, we won't match the `href` or `src` in a follow up tag.